cookbooks/aws-parallelcluster-environment/recipes/config/directory_service.rb

```diff
@@ -251,3 +251,14 @@ def parse_arn(arn_string)
     sensitive true
   end
 end unless on_docker?
+
+# TODO: add 'LoginNode' when 'domain_read_only_user' is added in the LoginNode dna.json
```

Comment:
We can remove this TODO as we are adding it in https://github.com/aws/aws-parallelcluster/pull/5519

Reply:
👍